spec/models/searchgov_url_spec.rb

```diff
@@ -113,22 +113,6 @@
         expect(fetch_required.pluck(:url)).
           to include 'https://www.agency.gov/crawled_more_than_month'
       end
-
-      it 'does not include urls last crawled more than 30 days ago and crawl status is not ok' do
```

Comment:
Why does this tests had to be removed?  The change affects queries using `enqueue` and `lastmod` but not `last_crawled_at` right? It seems that this functionality should still apply.